Orange/statistics/distribution.py

```diff
@@ -130,6 +130,19 @@ def from_data(cls, data, variable):
         self.variable = variable
         return self
 
+    @property
+    def array_with_unknowns(self):
+        """
+        This property returns a distribution array with unknowns arrays added
+        at the end
+
+        Returns
+        -------
+        np.array
+            Array with concatenated unknowns as a row and column
+        """
+        return np.append(np.array(self), self.unknowns)
```

Comment:
The documentation for append says it already works on the copy of the first arg - do you really need to make another copy? (or are you doing this to change it from a distribution to an array?)

Reply:
It just to transform to array from distribution. Otherwise, it stays distribution which is not good since we get another distribution with unknown attributes. In this case, the unknowns attribute is joined together with other data so it needs to be an array.